crates/turbopack/examples/turbopack.rs

```diff
@@ -64,12 +64,16 @@ async fn main() -> Result<()> {
                 .cell(),
                 Vc::cell("default".to_string()),
             );
-            let module = module_asset_context.process(
-                Vc::upcast(source),
-                Value::new(turbopack_core::reference_type::ReferenceType::Undefined),
-            );
-            let rebased = RebasedAsset::new(module, input, output);
-            emit_with_completion(Vc::upcast(rebased), output).await?;
+            if let Some(module) = *module_asset_context
```

Comment:
shouldn't this handle the none/error case as well?

Reply:
When the entry module resolves to nothing we don't have to emit anything